### PR TITLE
Adding hover effects to buttons 

### LINKED ISSUE #13: [Hover Effect] Hover effect on buttons on login and signup page
### Description

The current login page and signup page buttons don't have any buttons hover effect 
It would be visually appealing to users if we add hover effect on buttons 

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/Nishitbaria/OpenGrame/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)

### PR DESCRIPTION
**Pull request by Atharva1723**
## Changes proposed
The current sign in and sign up buttons doesn't have hover effect .. This pr adds hover effect

### CODE REVIEW

File: src/globals.css [hunk 1]
```diff
@@ -268,10 +268,12 @@
   .shad-textarea {
     @apply h-36 bg-dark-3 rounded-xl border-none focus-visible:ring-1 focus-visible:ring-offset-1 ring-offset-light-3 !important;
   }
-
+  .bg-darkpink-500 {
+    background-color: #d02d5e; /* Replace with your desired dark pink color */
```
Comment (Nishitbaria) [inline src/globals.css:272]: Hey please use tailwind css 

just write bg-#d02d5e direct instead of background-color : #d02d5e

**Comment by Atharva1723** [comment]
Should we add the color in the config ?

**Comment by Nishitbaria** [comment]
> Should we add the color in the config ?

I have make little bits changes according me

**Comment by Nishitbaria** [comment]
Thank you for your contribution 
@Atharva1723

**Comment by Atharva1723** [comment]
oh okay you updated the name accordingly

**Pull request merged**